compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/PlatformClipboard.desktop.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.platform
+
+import java.awt.Toolkit
+import java.awt.datatransfer.DataFlavor
+import java.awt.datatransfer.Transferable
+import java.awt.datatransfer.UnsupportedFlavorException
+
+actual typealias NativeClipboard = java.awt.datatransfer.Clipboard
```

Comment:
Exposing raw public typealias is a bad idea - it binds to AWT for desktop implementation without any options to compatibly change that.

Please consider changing that to extensions in platform sourcesets and removing this public API from common

Reply:
> cc @m-sasha Wdyt?

For the record, I disagree with the approach, but that is the approach we have chosen, so yes, we try to avoid attaching ourselves to AWT. But I would also add an extension method like this, in the desktop source-set:
```
@ExperimentalComposeUiApi
val Clipboard.awtClipboard: java.awt.datatransfer.Clipboard
    get() = nativeClipboard as java.awt.datatransfer.Clipboard
```